### PR TITLE
make xenial default now it has been released

### PR DESCRIPTION
**Pull request by mhilton**
Also make xenial boost higher in search. The search tests have been
refactored to make them less magic.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1746/
Test PASSed.

File: elasticsearch/elasticsearch.go [hunk 1]
```diff
@@ -337,14 +337,14 @@ func (db *Database) Search(index, type_ string, q QueryDSL) (SearchResult, error
 // marshaled as a json object and sent with the request. If v is non nil the response
 // body will be unmarshalled into the value it points to.
 func (db *Database) do(method, url string, body, v interface{}) error {
-	log.Debugf(">>> %s %s", method, url)
+	log.Tracef(">>> %s %s", method, url)
```
Comment (rogpeppe) [inline elasticsearch/elasticsearch.go:340]: Yes!

File: internal/storetesting/charm.go [hunk 2]
```diff
@@ -272,3 +272,23 @@ func MetaWithResources(m *charm.Meta, resources ...string) *charm.Meta {
 	}
 	return m
 }
+
+// MetaWithCategories returns m with Categories set to categories. If m
+// is nil, new(charm.Meta) will be used instead.
+func MetaWithCategories(m *charm.Meta, categories ...string) *charm.Meta {
```
Comment (rogpeppe) [inline internal/storetesting/charm.go:278]: It doesn't look like we're using either of these new functions.

**Comment by jrwren** [comment]
👍

File: internal/storetesting/charm.go (hunk 2 above)
Comment (mhilton) [inline internal/storetesting/charm.go:278]: We're not, they seem like they might be useful, but I'll remove them if you prefer.

**Comment by fabricematrat** [comment]
:+1:

**Comment by rogpeppe** [comment]
LGTM thanks

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**